doc/example/gradient_check.ipynb

```diff
@@ -861,7 +858,8 @@
     "- Find suitable simulation tolerances.\n",
     "- Consider switching from adjoint to forward sensitivities, which tend to be more robust.\n",
     "- Check the simulation logs for Warnings and Errors.\n",
-    "- Ensure that the model is correct."
+    "- Ensure that the model is correct.\n",
+    "- Be aware that, depending on the tolerances, the gradient might differ when it is calculated with sensitivities, because `Objective(x, sensi_orders=(0,))` provides a different value than `Objective(x, sensi_orders=(0,1))`"
```

Comment:
Personal preference, but I would formulate it a bit less strict. Currently, it sounds for me as if the two calls always provide different values.

Maybe you use can in the second part of the sentence.

Be aware that, depending on the tolerances, the gradients might differ when the objective function value is calculated with sensitivities, because `Objective(x, sensi_orders=(0,))`  can provide a different value than `Objective(x, sensi_orders=(0,1))`.